.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Validation
```

Comment:
| layout.tsx | in actions |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/1cab7f46-cc86-4184-b166-82ae5b40fff9) | ![image](https://github.com/user-attachments/assets/e4a192f5-e279-4fb6-93ce-18b49568d794) |